src/puppetlabs/puppetdb/command.clj

```diff
@@ -69,7 +69,7 @@
             [puppetlabs.puppetdb.utils :as utils]
             [slingshot.slingshot :refer [try+ throw+]]
             [puppetlabs.puppetdb.mq-listener :as mql]
-            [puppetlabs.puppetdb.command.constants :refer [command-names]]
+            [puppetlabs.puppetdb.command.constants :as cmd-consts]
```

Comment:
Yeah, I think it's reasonable to keep the :refer in here (esp. since command & command-constants are so closely related so the alias doesn't have much benefit), but the other ones that were already aliases I think should stay aliases.